bms-monitor.js

```diff
@@ -1,18 +1,44 @@
 const {expect} = require('chai');
 
-function batteryIsOk(temperature, soc, charge_rate) {
+function checkBatteryTemperature(temperature) {
+    console.log("checkBatteryTemperature: ",temperature)
     if (temperature < 0 || temperature > 45) {
-        console.log('Temperature is out of range!');
+        console.log('Temperature is out of range!', temperature);
         return false;
-    } else if (soc < 20 || soc > 80) {
+    }
+    
+        return true;
+    
+}
+function checkBatterySOC(soc) {
+    console.log("checkBatterySOC: ",soc)
+    if (soc < 20 || soc > 80) {
         console.log('State of Charge is out of range!')
         return false;
-    } else if (charge_rate > 0.8) {
+    } 
+   
+        return true;
+    
+}
+
+function checkBatteryChargeRate(charge_rate) {
+    console.log("checkBatteryChargeRate: ",charge_rate)
+    if (charge_rate<=0 || charge_rate > 0.8) {
         console.log('Charge rate is out of range!');
         return false;
     }
     return true;
 }
 
-expect(batteryIsOk(25, 70, 0.7)).to.be.true;
-expect(batteryIsOk(50, 85, 0)).to.be.false;
+expect(checkBatteryTemperature(25)).to.be.true;

```

Comment:
batteryIsOk is an API that is facade to the user. It abstracts the user from performing multiple calls. Is there any reason why that convenience is removed?